.rubocop.yml

```diff
@@ -54,3 +36,10 @@ Style/ClassAndModuleChildren:
 Naming/FileName:
   Exclude:
     - 'db/seeds.d/**/*'
+# Support both ruby19 and hash_rockets
+Style/HashSyntax:
+  Enabled: false
```

Comment:
Can we change this to `EnforcedStyle: no_mixed_keys` so you have to at least decide for one or the other syntax?

Reply:
@lzap: This is still missing. ;-)